smf/wicketd/manifest.xml

```diff
@@ -12,10 +12,25 @@
   </dependency>
 
   <exec_method type='method' name='start'
-      exec='ctrun -l child -o noorphan,regent /opt/oxide/wicketd/bin/wicketd run /var/svc/manifest/site/wicketd/config.toml --address %{config/address} --artifact-address %{config/artifact-address} --mgs-address %{config/mgs-address} --baseboard-file %{config/baseboard-file} &amp;'
+      exec='ctrun -l child -o noorphan,regent /opt/oxide/wicketd/bin/wicketd run /var/svc/manifest/site/wicketd/config.toml --address %{config/address} --artifact-address %{config/artifact-address} --mgs-address %{config/mgs-address} --nexus-proxy-address %{config/nexus-proxy-address} --baseboard-file %{config/baseboard-file} --read-smf-config &amp;'
     timeout_seconds='0' />
   <exec_method type='method' name='stop' exec=':kill' timeout_seconds='0' />
 
+  <!--
+    This refresh method only exists to support changing `config/rack-subnet`,
+    and makes a couple critical assumptions:
+
+      * `config/address` is not changing
+      * the dropshot server listening at `config/address` is not configured
+        for TLS
```

Comment:
We could validate these assumptions in code, within the body of `/reload-config`? It might be worthwhile to fail-hard in these cases if the error would otherwise be confusing.

Reply:
I don't think it's super easy to validate http vs https from within the endpoint, but that's maybe okay: the curl invocation itself will fail hard. I added a check that `config/address` is not changing in [53e0654](https://github.com/oxidecomputer/omicron/pull/4224/commits/53e06542b7c3b57722a12e672c35d49d3afb3d17)